llvm/include/llvm/IR/DataLayout.h

```diff
@@ -433,7 +434,10 @@ class DataLayout {
   ///
   /// For example, returns 40 for i36 and 80 for x86_fp80.
   TypeSize getTypeStoreSizeInBits(Type *Ty) const {
-    return 8 * getTypeStoreSize(Ty);
+    TypeSize BaseSize = getTypeSizeInBits(Ty);
+    uint64_t AlignedSizeInBits =
+        alignToPowerOf2(BaseSize.getKnownMinValue(), sizeof(uint64_t));
```

Comment:
```suggestion
        alignToPowerOf2(BaseSize.getKnownMinValue(), 8);
```
`sizeof(uint64_t)` doesn't have any relation to this. The 8 here is the number of bits in a byte.

Reply:
Leftover from a copy-paste, updated, thanks.